package.json

```diff
@@ -12,11 +12,19 @@
     "doc": "doc",
     "test": "test"
   },
+  "config": {
+    "docker": {
+      "tag": "",
+      "devuser": "asqhub",
+      "devrepo": "asq-dev"
+    }
+  },
   "scripts": {
     "test": "NODE_ENV=test mocha --reporter spec --bail --check-leaks test/unit/*",
     "start-dev": "NODE_ENV=development nodemon app.js dotenv_config_silent=true| bunyan",
     "start": "NODE_ENV=production node -r dotenv/config app.js dotenv_config_silent=true",
-    "install": "bower --allow-root cache clean && bower --allow-root install && CI_USER_TOKEN=${CI_USER_TOKEN} npm run ui && npm run cockpit && npm run plugins && npm run s2h-install && npm run pdf-bower-components",
+    "install-private-dep": "CI_USER_TOKEN=${CI_USER_TOKEN} npm run install-private-dependencies",
+    "install-external-dep": "bower --allow-root cache clean && bower --allow-root install && CI_USER_TOKEN=${CI_USER_TOKEN} npm run install-private-dependencies && npm run plugins && npm run pdf-bower-components && npm run live-app-bower-components",
```

Comment:
As you prefer, it actually depends if you are gonna actively use it or now, now that Wercker is there.